etcdserver/etcdserverpb/rpc.proto

```diff
@@ -469,6 +469,11 @@ message RoleRevokeRequest {
 
 message AuthEnableResponse {
   ResponseHeader header = 1;
+
+  enum Result {
```

Comment:
But I'll add an error code for representing a case of root account isn't created yet in the near future. I thought adding the return code for the success now -> adding the error code later would produce more reader friendly git log. How do you think?